lib/install.js

```diff
@@ -32,6 +41,15 @@ function install(opts, cb) {
                 return cb1();
             });
 
+            if (opts.enroute.hotReload) {
+                if (typeof opts.basePath !== 'undefined') {
```

Comment:
@paulbakker one qq - in what cases was basePath not defined?

Reply:
It was an assumption that it’s possibly not defined. If this option is always required we can go without the check.

Reply:
Cool thanks! It's always required, so I think we can simply the check. 👍